### ISSUE TITLE
Footnotes link to full page path (instead of internal link)

### ISSUE THREAD

**Issue by lhagan** [rcarmo/Yaki#5]
For example:

```
<a href="/space/blog/2012/03/17/1802#fn:1" rel="/space/blog/2012/03/17/1802#fn:1" class="anchor" title="link to fn:1 in this page">1</a>
```

Should be:

```
<a href="#fn:1" rel="/space/blog/2012/03/17/1802#fn:1" class="anchor" title="link to fn:1 in this page">1</a>
```

This means an unnecessary page load for the user and also prevents things like [Footnotify](http://openideas.ideon.co/2011/introducing-footnotify-extension) from working properly.

**Comment by rcarmo** [comment]
Correct. That is due to the case where you have multiple articles on a page with footnotes. Changing the behavior will break that use case, but I'll consider it.

**Comment by lhagan** [comment]
Right, I think that's an important use case. DF adds the date to the footnote link, presumably to avoid this issue:

```
<a href="#fn1-2012-03-14">1</a>
```

Perhaps a better option would be to append the path to the wiki node:

```
<a href="#fn1-blog-2012-03-17-1802">1</a>
```

**Comment by rcarmo** [comment]
Quite. Still, I think that will involve wrapping the markup renderers, and besides Markdown, the others have different footnote outputs and whatnot.. .Might be a pain.
